src/styles/main.scss

```diff
@@ -0,0 +1,58 @@
+@import '../styles/variables';
+
+body {
+  margin: 0;
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: $font-family;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: calc(7 * ($day-size + 1px));
```

Comment:
The calculation within `calc` function is not following the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). There should be spaces around the `+` and `-` operators.